README.md

```diff
@@ -1,39 +1,58 @@
-# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=master)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
-Kubernetes deployment of RADAR-base.
+# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=dev)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
+The repository contains Helm charts and other configuration needed to install RADAR-base on a Kubernetes cluster.
 
+## Prerequisites
+#### Kubernetes cluster 
```

Comment:
Please use headers in order
```
# H1
## H2
### H3
```
It's good practice to always add some proper content between one header and the next.